.buildkite/x-pack/pipeline.xpack.metricbeat.yml

```diff
@@ -245,56 +248,22 @@ steps:
           - "x-pack/metricbeat/build/*.json"
         notify:
           - github_commit_status:
-              context: "x-pack/metricbeat: MacOS arm64 Unit Tests"
+              context: "x-pack/metricbeat: macOS arm64 Unit Tests"
 
-      - label: ":linux: Cloud (MODULE) Tests"
+      - label: ":ubuntu: x-pack/metricbeat AWS Tests"
         key: "x-pack-metricbeat-extended-cloud-test"
         if: build.env("GITHUB_PR_LABELS") =~ /.*aws.*/
         command: |
           set -euo pipefail
-          # defines the MODULE env var based on what's changed in a PR
-          source .buildkite/scripts/changesets.sh
-          defineModuleFromTheChangeSet x-pack/metricbeat
-          echo "~~~ Running tests"
-          source .buildkite/scripts/setup_cloud_env.sh
-          cd x-pack/metricbeat
-          mage build test
-        env:
-          ASDF_TERRAFORM_VERSION: 1.0.2
-          AWS_REGION: "eu-central-1"
-          MODULE_DIR: x-pack/metricbeat/module/aws
-          REPO: beats
-        agents:
-          provider: "gcp"
-          image: "${IMAGE_UBUNTU_X86_64}"
-          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
-        artifact_paths:
-          - "x-pack/metricbeat/build/*.xml"
-          - "x-pack/metricbeat/build/*.json"
-        notify:
-          - github_commit_status:
-              context: "x-pack/metricbeat: Cloud (MODULE) Tests"
-
-      - label: ":linux: Cloud AWS (MODULE) Tests"
-        key: "x-pack-metricbeat-extended-cloud-test-aws"
-        skip: "Skipping due elastic/beats#36425 & elastic/ingest-dev#3170"
-        # https://github.com/elastic/beats/issues/36425 & https://github.com/elastic/ingest-dev/issues/3170
-        if: build.env("GITHUB_PR_LABELS") =~ /.*aws.*/
-        command: |
-          set -euo pipefail
-          # defines the MODULE env var based on what's changed in a PR
-          source .buildkite/scripts/changesets.sh
-          defineModuleFromTheChangeSet x-pack/metricbeat
-
+          source .buildkite/scripts/initCloudEnv.sh
           echo "~~~ Running tests"
-          source .buildkite/scripts/setup_cloud_env.sh
           cd x-pack/metricbeat
           mage build test goIntegTest
         env:
           ASDF_TERRAFORM_VERSION: 1.0.2
-          AWS_REGION: "eu-central-1"
-          MODULE_DIR: x-pack/metricbeat/module/aws
-          REPO: beats
+          MODULE_DIR: "x-pack/metricbeat/module/aws"
+          MODULE: "aws"
```

Comment:
If we are setting `aws` in TEST_TAGS: https://github.com/elastic/beats/pull/39742/files#diff-e05866df538e5a55b5d3b2539addc586af1a53748087ac840220e7e666ae7066R17

do we still need to define it in MODULE too? Because from what I remember we had some script that went through definitions in MODULE and applied them in TEST_TAGS accordingly but things may have changed.

Reply:
Test tags seems to be not affecting anything, I've just raised an issue for that: https://github.com/elastic/ingest-dev/issues/3476. So we need to have MODULE definition for sure, otherwise all tests (not only cloud) are being executed on week EC2 instance and fail on timeout.